site/content/docs/latest/tutorials/getting-started.md

```diff
@@ -48,7 +58,7 @@ kubectl get --namespace default secret $(kubectl get --namespace default service
 Open a Powershell terminal and run:
 
 ```powershell
-[Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($(kubectl get --namespace default secret $(kubectl get --namespace default serviceaccount kubeapps-operator -o jsonpath='{.secrets[].name}') -o jsonpath='{.data.token}')))
+[Text.Encoding]::UTF8.GetString([System.Convert]::FromBase64String($(kubectl get --namespace default secret kubeapps-operator-token -o jsonpath='{.data.token}')))
```

Comment:
I'll give it a try soon just to double-check it works

Reply:
Thanks for checking the powershell :)